source/includes/options-mongodump.yaml

```diff
@@ -252,6 +252,19 @@ description: |
   :term:`master` nodes in master/slave replication deployments.
 
   {{role}} does not dump the oplog collection.
+
+  .. note::
+
+     To use :program:`mongodump` with {{role}}, you must create a full
+     dump of a :term:`replica set` member. :program:`mongodump` with 
+     {{role}} fails if you use any of the following options along with 
+     {{role}} to limit the data to be dumped:
```

Comment:
You already stated that mongodump with {{role}} fails, so you don't need to end the sentence with along with {{role}}

Note also, that the along with {{role}} at the end makes it sound like you are using oplogReplay to also limit the data to be dumped.

Reply:
Argh. Fixed.